logback-core/src/test/java/ch/qos/logback/core/BasicStatusManagerTest.java

```diff
@@ -87,5 +87,14 @@ public void duplicateInstallationsOfOnConsoleListener() {
     }
   }
 
-
+  @Test
+  public void testFireStatusAddEventConcurrentModificationException() {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [JUnit tests should include assert() or fail()](https://www.codacy.com/app/tony19/logback/file/2371722102/issues/source?bid=3222618&fileBranchId=3222683#l91)